src/plugins/blocksmith/components/blocksmith/Blocksmith.jsx

```diff
@@ -1356,11 +1372,36 @@ class Blocksmith extends React.Component {
   }
 
   /**
-   * If there is a copied block available in redux, use it to create a draft block with it as the
-   * data payload.
+   * Dereferences any nodes in the copied block that are referenced
+   */
+  derefCopiedBlockNodes() {
```

Comment:
@bhsiao55 cant you do this only on paste? or, even better, put into the placeholder components themselves so blocksmith doesnt need more overhead

Reply:
@efjacobson I tried calling derefCopiedBlockNodes on paste but for example when pasting an article block the title and image would render at different times so to eliminate that effect I called it in componentDidUpdate.  If I placed it in the individual placeholder components I suspect it will have that same effect.

Reply:
@bhsiao55 its only a very short time though eh? why not just not render the placeholder until the title/image are available?

Reply:
@efjacobson yea it's for a short time.  ok I will move the loading of the nodes to the preview component